CHANGES.rst

```diff
@@ -24,12 +24,12 @@ For plans and progress see https://github.com/gammapy/gammapy/milestones/0.4
 Contributors
 ++++++++++++
 
-- ...
+- Axel Donath
 
 Pull requests
 +++++++++++++
 
-- ...
+-  Fixed TS map boundary handling [#332] (Axel Donath) 
```

Comment:
Thanks for adding a changelog entry. It's really minor and nitpicky, but for the future: please use active voice in GH issues / commits / changelog entries, i.e. "Fix" instead of "Fixed".